table/table_reader_bench.cc

```diff
@@ -100,7 +100,7 @@ void TableReaderBenchmark(Options& opts, EnvOptions& env_options,
     tb = opts.table_factory->NewTableBuilder(
         TableBuilderOptions(
             ioptions, moptions, ikc, &int_tbl_prop_collector_factories,
-            CompressionType::kNoCompression, CompressionOptions(),
+            CompressionType::kNoCompression, 0, CompressionOptions(),
```

Comment:
Add a comment for `0`, i.e. `0 /* sample_for_compression */`? This applies to a few other places in this PR as well.

Reply:
Fixed.